static/app/views/alerts/list/rules/teamFilter.tsx

```diff
@@ -1,119 +1,120 @@
-import {useState} from 'react';
+import {Fragment, useMemo} from 'react';
 import styled from '@emotion/styled';
 import debounce from 'lodash/debounce';
 
-import Input from 'sentry/components/forms/controls/input';
-import LoadingIndicator from 'sentry/components/loadingIndicator';
+import TeamAvatar from 'sentry/components/avatar/teamAvatar';
+import Badge from 'sentry/components/badge';
+import CompactSelect from 'sentry/components/forms/compactSelect';
 import {DEFAULT_DEBOUNCE_DURATION} from 'sentry/constants';
+import {IconUser} from 'sentry/icons';
 import {t} from 'sentry/locale';
-import space from 'sentry/styles/space';
-import {isActiveSuperuser} from 'sentry/utils/isActiveSuperuser';
 import useTeams from 'sentry/utils/useTeams';
 
-import Filter from './filter';
-
 interface Props {
-  handleChangeFilter: (activeFilters: Set<string>) => void;
-  selectedTeams: Set<string>;
-  fullWidth?: boolean;
+  handleChangeFilter: (activeFilters: string[]) => void;
+  selectedTeams: string[];
```

Comment:
Making `TeamFilter` work with arrays instead of sets. Currently, we convert sets to arrays and vice versa. This back and forth conversion seems inefficient, and I don't know what the benefit of using sets is in this case (the only one I can think of is that sets have unique members, but given the way we implement this filter, uniqueness is pretty much guaranteed).